src/sentry/search/events/fields.py

```diff
@@ -534,6 +542,7 @@ def resolve_orderby(orderby, fields, aggregations):
     those that are currently selected.
     """
     orderby = orderby if isinstance(orderby, (list, tuple)) else [orderby]
+    equation_aliases = [equation[-1] for equation in equations]
```

Comment:
Looks like `equations` can be `None` here because it's a default arg from `resolve_field_list`.